### PR TITLE
cluster-api-provider-openstack: rename default branch to main

### PR DESCRIPTION
**Pull request by tobiasgiese**
follow-up of #24145 

/cc @sbueringer 

<sub>Tobias Giese <tobias.giese@daimler.com>, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sub>

### CODE REVIEW

File: config/jobs/image-pushing/k8s-staging-cluster-api.yaml [hunk 1]
```diff
@@ -265,7 +265,7 @@ periodics:
     extra_refs:
       - org: kubernetes-sigs
         repo: cluster-api-provider-openstack
-        base_ref: master
+        base_ref: main
         path_alias: sigs.k8s.io/cluster-api-provider-openstack
     spec:
       serviceAccountName: gcb-builder
```
Comment (sbueringer) [inline config/jobs/image-pushing/k8s-staging-cluster-api.yaml:271]: There's one more in l.287

Comment (tobiasgiese) [reply config/jobs/image-pushing/k8s-staging-cluster-api.yaml:271]: Thanks 👍🏻

**Comment by sbueringer** [comment]
/lgtm

/assign @dims 
Can you please approve?

**Pull request merged**